fire/cli/niv/_udtræk_revision.py

```diff
@@ -52,8 +51,9 @@ def udtræk_revision(
         "IDENT:refgeo_id",
         "IDENT:station",
         "NET:10KM",
-        "SKITSE:md5",
-        "ATTR:fundamentalpunkt",
+        "SKITSE:master_md5",
+        "SKITSE:master_sti" "SKITSE:png_md5",
```

Comment:
Der mangler vist et par kommaer i disse to linjer

Reply:
Godt set!

Reply:
Syntaktisk er det jo korrekt nok, men vi kommer nok aldrig til at benytte attributten `SKITSE:master_stiSKITSE:png_md5` :-)

Reply:
Nej, bliver noget værre rod :)